code/datums/mutations/hulk.dm

```diff
@@ -28,7 +28,8 @@
 	if(source.a_intent != INTENT_HARM)
 		return
 	if(target.attack_hulk(owner))
-		source.say(pick(";RAAAAAAAARGH!", ";HNNNNNNNNNGGGGGGH!", ";GWAAAAAAAARRRHHH!", "NNNNNNNNGGGGGGGGHH!", ";AAAAAAARRRGH!" ), forced="hulk")
+		if(world.time < 20)
```

Comment:
Yes, this will actually stops hulk screaming for the rest of the round past the first 2 ingame seconds after the server is booted.
I guess you could either make this a prob() or add a new var to the mutation to keep track of the delay.

Reply:
Huh? Well I've tested this multiple times on locally and it worked fine. I guess I'll add a var then if that don't work